lxd/resource_lxd_container.go

```diff
@@ -204,6 +204,13 @@ func resourceLxdContainer() *schema.Resource {
 				Default:  true,
 				ForceNew: false,
 			},
+
+			"project": {
+				Description: "The project where the container will be created. Defaults to the provider's project, or 'default'.",
```

Comment:
nit: let's remove the description so there's consistency with the other parameters. The only area where parameters have descriptions are in the provider.